source/zh-tw/docs/github-pages.md

```diff
@@ -32,45 +33,58 @@ deploy:
 
 ## Private repository
 
-This section only applies to private repo.
+由於在 Private Repository 啟用 Travis CI 需要付費，因此妳可能需要自己構建。
 
-1. Create a repo named <b>*username*.github.io</b>, where username is your username on GitHub. If you have already uploaded to other repo, rename the repo instead. _(Skip to step 3 if you prefer not to upload your source folder to GitHub at all)_
-2. Push the files of your Hexo folder to the repository. The `public/` folder is not (and should not be) uploaded by default, make sure the `.gitignore` file contains `public/` line. The folder structure should be roughly similar to [this repo](https://github.com/hexojs/hexo-starter), without the `.gitmodules` file.
-3. Run `hexo generate` and copy the `public/` folder to somewhere else (in your workstation).
-4. Create a new `gh-pages` git branch from your Hexo folder, we recommend creating an orphan branch (to create a new branch without commit history):
-``` bash
-$ git checkout --orphan gh-pages
+1. 新建壹個 repository。如果妳希望妳的站點能通過 `<妳的 GitHub 用戶名>.github.io` 域名訪問，妳的 repository 應該直接命名為 `<妳的 GitHub 用戶名>.github.io`。如果妳不想上傳妳的 Hexo 站點文件夾到 GitHub 請直接跳到第三步。
+2. 將妳的 Hexo 站點文件夾推送到 GitHub，默認情況下不應該 `public` 目錄將不會被推送到 repository 中，妳應該檢查 `.gitignore` 文件中是否包含 `public` 壹行，如果沒有請加上。
+3. 執行 `hexo g` 生成站點目錄，並將生成的 `public` 復制到別處。
+4. 在 Hexo 站點目錄下新建壹個 `gh-pages` 分支，我們推薦創建壹個 orphan 分支（壹個幹凈的、沒有任何 commit 歷史的分支）：
+
+```bash
+$ git checkoud --orphan gh-pages
 ```
-5. Remove everything including any hidden files **except** the `.git/` folder. Fret not, those files are still residing at the master branch.
-6. Move the _content_ of the `public/` folder back.
-7. Commit and push the gh-pages branch.
+
+5. 將 Hexo 站點文件夾下除了 `.git` 以外的任何文件和目錄全部刪除（不要擔心，它們還能被找回來的）
+6. 找到之前復制到別處的的 `public` 目錄，將其中的的所有文件復制回 Hexo 站點文件夾。
+7. 使用以下指令發布站點
+
 ``` bash
 $ git add .
 $ git commit -a -m "Initial commit"
 $ git push origin gh-pages
 ```
-8. In your repo's setting, navigate to "GitHub Pages" section and change Source is **gh-pages branch**.
-9. Check the webpage at your-username.github.io.
-10. To navigate back to your source folder,
+
+8. 前往妳的 GitHub 的 repository 設置頁面，將 **GitHub Pages** 的部署分支設置為 `gh-paghes` 分支。
+9. 前往 `https://<妳的 GitHub 用戶名>.github.io` 查看妳的站點是否可以訪問。這可能需要壹些時間。
+10. 回到妳的 Hexo 站點文件夾，使用下面指令
+
 ``` bash
 $ git checkout master
 ```
 
-## Useful links
+11. 妳的 Hexo 站點文件就回來了。
+
+{% note info Windows 用戶 %}
+請註意，以上教程翻譯自 Hexo 文檔英文版，僅做參考；譯者在翻譯過程中發現很多語焉不詳的內容。
+在 [Awesome Hexo](https://github.com/hexojs/awesome-hexo) 中收錄有壹些在 GitHub 部署 Hexo 的教程。通過搜索引擎可以找到更多教程。
+也歡迎更多有誌之士前來改善 Hexo 文檔，不勝感激。
+{% endnote %}
+
+## 有用的參考鏈接
 
-- [GitHub Pages](https://help.github.com/categories/github-pages-basics/)
-- [Travis CI Docs](https://docs.travis-ci.com/user/tutorial/)
+- [GitHub Pages 使用文檔](https://help.github.com/categories/github-pages-basics/)
+- [Travis CI 使用文檔](https://docs.travis-ci.com/user/tutorial/)
+- [Awesome Hexo](https://github.com/hexojs/awesome-hexo)
+- [在百度上搜索 "Hexo GitHub"](https://www.baidu.com/s?wd=Hexo%20GitHub)
```

Comment:
IMHO, since this is a zh-TW version, I will change this to Google instead of Baidu.

Reply:
Agree.